.env.example

```
@@ -60,13 +60,16 @@ YOU_API_KEY=
 PARALLEL_API_KEY=
 SERPER_API_KEY=
 ANTHROPIC_AUTH_TOKEN=
+OPENAI_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OPENAI_API_KEY key should go before the PARALLEL_API_KEY key

(UnorderedKey)